examples/provider_store/README.md

```
@@ -39,6 +39,11 @@ INFO:  			/pub/robotics/parts/gobilda:hardware/nut_m4_0_7mm#1
 ...
 ```
 
+Confirm it works for aliases just fine:
+```shell
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)